resources/views/layouts/errors.blade.php

```diff
@@ -0,0 +1,21 @@
+@if (count($errors))
+
+  <div class="form-group">
+
```

Comment:
having a new line in between HTML within the same block(if that makes sense) looks a little off to me, but other than that looks good